packages/formatter-json/README.md

```diff
@@ -0,0 +1,14 @@
+# Formatter json (`@sonarwhal/formatter-json`)
```

Comment:
With rules we [only specified the name of the rule between `()`](https://github.com/sonarwhal/sonarwhal/tree/master/packages/rule-apple-touch-icons#require-an-apple-touch-icon-apple-touch-icons). We should be consistent and either use only the name here, or update things everywhere.

Reply:
I think we should use the name of the package between `()`

Reply:
Can you also update the rules then? Thanks!